lib/internal/Magento/Framework/ObjectManager/Code/Generator/Repository.php

```diff
@@ -228,7 +228,6 @@ protected function _getGetMethod()
             'parameters' => [
                 [
                     'name' => 'id',
-                    'type' => 'int',
```

Comment:
This will cause an exception if someone uses type hinting in a repository interface.